src/hooks/useProductList.js

```diff
@@ -1,13 +1,18 @@
 import dayjs from 'dayjs'
 
+import firstProductUrl from '../assets/images/product1.svg'
+import secondProductUrl from '../assets/images/product2.svg'
+import thirdProductUrl from '../assets/images/product3.svg'
+import fourProductUrl from '../assets/images/product4.svg'
```

Comment:
I know they are fake data but I would rename them to be more representative of the products (and also move them inside `images/fake_products/`. It's a little detail but it can help someone else understand how the code or data representation maps to the concrete UI. It's not a required change, so don't worry too much.

Reply:
Do rename must be for example 'bottleProduct'?